File: mobileAppClient/mobileAppClient/Models/Hospital.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace mobileAppClient.Models
{
    /*
     * Class which holds information about a helicopter on the map
     */
    public class Hospital
    {
        public string name { get; set; }
        public string address { get; set; }
        public string region { get; set; }
        public string country { get; set; }
        public double latitude { get; set; }
        public double longitude { get; set; }
    }
}

File: server/src/main/java/seng302/Model/Hospital.java
package seng302.Model;

public class Hospital {

    private String name;
    private String address;
    private String region;
    private String city;
    private String country;
    private double latitude;
    private double longitude;

    public Hospital(String name, String address, String region, String city, String country, double latitude, double longitude){
        this.name = name;
        this.address = address;
        this.region = region;
        this.city = city;
        this.country = country;
        this.latitude = latitude;
        this.longitude = longitude;
    }
}
